src/__tests__/components/helpers/breadcrumb.spec.js

```diff
@@ -17,70 +17,59 @@ describe('RequestInformation', () => {
     expect(wrapper.type()).toEqual(Grid);
   });
 
-  describe(' Row', () => {
-    let row;
+  describe('Step 1', () => {
```

Comment:
All of these `Step` tests look the same, so it would be worth looking into [`describe.each`](https://jestjs.io/docs/en/api#describeeachtable-name-fn-timeout)